monkey/infection_monkey/network/info.py

```diff
@@ -124,14 +132,18 @@ def get_free_tcp_port(min_range=1000, max_range=65535):
 
 def check_internet_access(services):
     """
-    Checks if any of the services are accessible, over ICMP
+    Checks if any of the services are accessible, over HTTPS
     :param services: List of IPs/hostnames
     :return: boolean depending on internet access
     """
-    ping_str = "-n 1" if sys.platform.startswith("win") else "-c 1"
     for host in services:
-        if os.system("ping " + ping_str + " " + host) == 0:
+        try:
+            requests.get("https://%s" % (host,), timeout=TIMEOUT)
```

Comment:
Is that endpoint signed? Who signs it? Are we sure we want to rely on the certificate being valid?
If not, lets add a verify=false or something

Reply:
It should be signed, but since this is only for internet check we can add verify=False